lib/zipkin-tracer/trace_client.rb

```diff
@@ -5,19 +5,22 @@ class TraceClient
 
     def self.local_component_span(local_component_value, &block)
       if block_given?
```

Comment:
why is this `if` needed? the block is in the args, if someone calls it without providing it an exception would be raised:
```ruby
def a(b, &block)
  puts b
  yield
end

a("this is b") { puts "from block" }
# => this is b
# => from block

a("this is b")
#=> this is b
#=> LocalJumpError: no block given (yield)
```